templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver?</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+
+    <button type="submit" name="confirm" class="btn btn-danger">Yes</button>
+    <button type="submit" name="cancel" class="btn btn-secondary">Cancel</button>
```

Comment:
try this one

`<a href="{% url 'taxi:driver-detail' pk=driver.id %}" class="btn btn-secondary">Cancel</a>`